src/main/java/com/bookshop/repository/cart/CartRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.bookshop.repository.cart;
+
+import com.bookshop.model.ShoppingCart;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CartRepository extends JpaRepository<ShoppingCart, Long> {
+    ShoppingCart getShoppingCartByUser_Id(Long id);
```

Comment:
Is it possible to use Java naming convention?

Reply:
That name is generated in interface, I don't think it will work if I try to change it